Lessen_6_hw/task_2.js

```diff
@@ -0,0 +1,90 @@
+/*
+1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+  и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+  Пиццы конкурента:
+  const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+*/
+
+function getEmployeeInfo(pizza) {
+	const competitorPizzas = [
+		'Peperoni',
+		'Caprichosa',
+		'Diablo',
+		'4 cheeses',
+		'hawai',
+	]
+	let unique = []
+	for (let i = 1; i < pizza.length; i++) {
```

Comment:
Start with `0` index, otherwise you lost first element of an array
```suggestion
	for (let i = 0; i < pizza.length; i++) {
```